src/Adapters/LaravelCacheAdapter.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+namespace Spatie\Prometheus\Adapters;
+
+use Illuminate\Contracts\Cache\Repository;
+use Prometheus\Counter;
+use Prometheus\Gauge;
+use Prometheus\Histogram;
+use Prometheus\Storage\InMemory;
+use Prometheus\Summary;
+
+class LaravelCacheAdapter extends InMemory
+{
+    private const CACHE_KEY_PREFIX = 'PROMETHEUS_';
+
+    private const CACHE_KEY_SUFFIX = '_METRICS';
+
+    private const STORES = [Gauge::TYPE, Counter::TYPE, Histogram::TYPE, Summary::TYPE];
+
+    public function __construct(private readonly Repository $cache)
+    {
+    }
+
+    public function collect(bool $sortMetrics = true): array
```

Comment:
Could you add a doc block that also has info on the content of the returned array?